overlays/openstack.yaml

```diff
@@ -0,0 +1,18 @@
+applications:
+  openstack-integrator:
+    charm: openstack-integrator
+    num_units: 1
+    trust: true
+    base: ubuntu@22.04
+  openstack-cloud-controller:
+    charm: openstack-cloud-controller
+    base: ubuntu@22.04
+  cinder-csi:
+    charm: cinder-csi
+    base: ubuntu@22.04
+relations:
+  - [openstack-cloud-controller:kube-control,            k8s:kube-control]
+  - [openstack-cloud-controller:external-cloud-provider, k8s:external-cloud-provider]
+  - [openstack-cloud-controller:openstack,               openstack-integrator:clients]
+  - [easyrsa:client,                                     cinder-csi:certificates]
```

Comment:
BURN it 🗡️ 

But i think cinder-csi will need to relate to kube-control

Reply:
to use cinder-csi with kube-control, we'll need this merged
* https://github.com/canonical/cinder-csi-operator/pull/9

Reply:
Forgot this one, my bad.